tony-core/src/main/java/com/linkedin/tony/rpc/impl/ApplicationRpcClient.java

```diff
@@ -77,19 +77,14 @@ private ApplicationRpcClient(String serverAddress, int serverPort, Configuration
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
-    RetryPolicy retryPolicy;
+    RetryPolicy retryPolicy = RetryPolicies.exponentialBackoffRetry(10, 1000, TimeUnit.MILLISECONDS);
```

Comment:
Any reason for exponential retry here? The default should be retryUpToMaximumTimeWithFixedSleep